airflow/lineage/datasets.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we split the different files to different files? I maybe move these files in the dataset module.

Reply:
I think that is a bit over kill. At the moment datasets are designed to be light weight so it doesnt make much sense to split it out right now.